apps/civicsignalblog/src/components/Author/Author.snap.js

```diff
@@ -11,7 +11,7 @@ exports[`<Author /> renders unchanged 1`] = `
       Article by
     </div>
     <div
-      class="MuiStack-root css-3oepjh-MuiStack-root"
```

Comment:
Is there a way to only have these generate at build time? I think having them in source adds to the number of changes giving false positives.

Reply:
https://jestjs.io/docs/snapshot-testing#snapshot-testing-with-jest